coordinator/src/test/java/org/apache/uniffle/coordinator/strategy/storage/LowestIOSampleCostSelectStorageStrategyTest.java

```diff
@@ -103,19 +83,19 @@ public void selectStorageTest() throws Exception {
     assertEquals(0.0, CoordinatorMetrics.GAUGE_USED_REMOTE_STORAGE.get(storageHost1).get(), 0.5);
     String storageHost2 = "p2";
     assertEquals(0.0, CoordinatorMetrics.GAUGE_USED_REMOTE_STORAGE.get(storageHost2).get(), 0.5);
+    applicationManager.getSelectStorageStrategy().detectStorage();
+    assertEquals(0, applicationManager.getRemoteStoragePathRankValue().get(remoteStorage1).getAppNum().get());
+    assertEquals(0, applicationManager.getRemoteStoragePathRankValue().get(remoteStorage2).getAppNum().get());
 
     // compare with two remote path
     applicationManager.incRemoteStorageCounter(remoteStorage1);
     applicationManager.incRemoteStorageCounter(remoteStorage1);
     String testApp1 = "application_test_" + 1;
     applicationManager.registerApplicationInfo(testApp1, "user");
     applicationManager.refreshAppId(testApp1);
+    selectStorageStrategy.sortPathByRankValue(remoteStorage2, testFile, System.currentTimeMillis());
     Thread.sleep(1000);
-    final long current = System.currentTimeMillis();
-    fs.create(path);
-    selectStorageStrategy.sortPathByRankValue(remoteStorage2, testFile, current);
-    fs.create(path);
-    selectStorageStrategy.sortPathByRankValue(remoteStorage1, testFile, current);
+    selectStorageStrategy.sortPathByRankValue(remoteStorage1, testFile, System.currentTimeMillis());
```

Comment:
Why do we need to sleep before this line?

Reply:
Ensure that the `System.currentTimeMillis()` corresponding to `remoteStorage1` is greater than that of `remoteStorage2`, because under the `IO` policy, this time is used to measure which remote path is selected, and the smaller the time, the better it will be selected.

Reply:
This sleep time can actually be set smaller.

Reply:
> Ensure that the `System.currentTimeMillis()` corresponding to `remoteStorage1` is greater than that of `remoteStorage2`, because under the `IO` policy, this time is used to measure which remote path is selected, and the smaller the time, the better it will be selected.

Let's add this comment to the code?

Reply:
OK.

Reply:
OK.